projects/experimental/directives/card/card.styles.less

```diff
@@ -0,0 +1,57 @@
+@import 'taiga-ui-local';
+
+[tuiCard] {
+    .clearbtn();
+    display: flex;
+    background: var(--tui-base-02);
+    border-radius: 0.75rem;
+    box-sizing: border-box;
+    padding: 0.75rem;
+    gap: 0.75rem;
+    flex-direction: column;
+    text-decoration: none;
+
+    &[data-size='m'] {
+        width: 8.75rem;
+        height: 8.75rem;
+    }
+
+    &:not([data-platform='android']) {
+        transition: transform 100ms ease-in;
+    }
+
+    &._touched[data-platform='ios'] {
+        transform: scale(0.95);
+    }
+
+    &[data-platform='web']:hover {
+        transform: scale(1.15);
+    }
+
+    & > * {
+        overflow: hidden;
+    }
+
+    & > :nth-child(2),
+    [tuiCardBottom] {
+        margin-top: auto;
+        max-height: 50%;
+    }
+
+    [tuiTitle] {
+        font-weight: bold;
+    }
+
+    [tuiSubtitle] {
+        color: var(--tui-text-01);
+        font-weight: normal;
+    }
+
+    tui-badge {
+        width: fit-content;
```

Comment:
Add `align-items: flex-start;` on host instead.

Reply:
This breaks current examples of scrollbars, but you can just add max-width: 100% on them. Also there's vertical scrolling on the label here:
<img width="344" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/11832552/d6c9b563-8e30-462f-8206-c81e8cf82965">

Reply:
fixed